code/modules/clothing/head/helmet.dm

```diff
@@ -379,7 +379,7 @@
 	desc = "An intimidating tribal helmet, it doesn't look very comfortable."
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDESNOUT
 	flags_cover = HEADCOVERSEYES
-	armor = list(MELEE = 35, BULLET = 25, LASER = 25, ENERGY = 35, BOMB = 25, BIO = 0, FIRE = 50, ACID = 50)
+	armor = list(MELEE = 35, BULLET = 25, LASER = 25, ENERGY = 35, BOMB = 40, BIO = 0, FIRE = 50, ACID = 50) //SKYRAT EDIT; Original BOMB = 25
```

Comment:
Bone armor was already strong, with it's only weakness being the KPA and plasma cutters, buffing it against one, then nerfing the other entirely is the definition of powercreep.

Reply:
Oh, right, I forgot me nerfing cutters.